src/Microsoft.AspNet.Razor/TagHelpers/TagHelperAttributeDescriptor.cs

```diff
@@ -116,7 +117,6 @@ public string TypeName
                 }
 
                 _typeName = value;
-                IsStringProperty = string.Equals(_typeName, typeof(string).FullName, StringComparison.Ordinal);
```

Comment:
Why? I think this change would actually break back compatibility within the `TagHelperDescriptor` resolution that we use at design time. For instance:

Old `Razor.Tooling` (`isStringProperty` is not serialized across the wire) + new VisualStudio = VS trying to deserialize the payload into the new `TagHelperAttributeDescriptor` format would result in `IsStringProperty` not being set (unless VS hardcoded that logic on their end, badd).